### DISCUSSION TITLE
Starter kits + Community starter kits

### DISCUSSION THREAD

**Discussion by GabrielSieben** [livewire/flux#149] [Feature Requests]
Hello,

Perhaps in the wiki, or elsewhere, we could make a list of community starter kits because Flux does not have one yet?

I know that @joshcirre is working on “fission”, a starter kit based on a Volt+Folio+Flux stack, at https://github.com/joshcirre/fission. I recently finished my own port of Jetstream at https://github.com/gjsman/fluent, with a Jetstream+Folio version at https://github.com/gjsman/fluent-folio. Until there is an official starter kit, a “Community Starter Kits” or similar heading in the readme or docs may be useful.

**Comment by calebporzio** [comment]
Closing this as Laravel added one: https://laravel.com/docs/12.x/starter-kits#livewire

We should add a section to the docs somewhere pointing to it as well though...
